va-gov/components/react-loader.html

```diff
@@ -0,0 +1,23 @@
+{% if !loadingMessage %} {% assign loadingMessage = 'Please wait while we load the application for you.' %} {% endif %}
+{% if in_maintenance == true %}
+  <div class="row">
+      <div class="small-12 columns">
+        <div class="csp-inline-patch-page-react">
+        <h3>{{ maintenance_line1 }}</h3>
+        <h4>{{ maintenance_line2 }}</h4>
+        <a href="/"><button>Go back to Vets.gov</button></a>
+        </div>
+      </div>
+    </div>
+{% else %}
+  {{ contents }}
+  <div class="section">
+    <div id="react-root">
+      <div class="loading-indicator-container">
+        <div class="loading-indicator" role="progressbar" aria-valuetext="{{ loadingMessage }}" tabIndex="0">
+        </div>
+        {{ loadingMessage }}
+      </div>
+    </div>
+  </div>
+{% endif %}
```

Comment:
Not sure of a great name for this, but this is ultimately the kickoff for React applications.